Framework/CMakeLists.txt

```diff
@@ -70,6 +74,9 @@ add_subdirectory (Types)
 include_directories (Kernel/inc)
 add_subdirectory (Kernel)
 
+include_directories (Parallel/inc)
+add_subdirectory (Parallel)
```

Comment:
Wouldn't it be better to wrap this in the condition `if (MPI_EXPERIMENTAL)` rather than the large block condition in `Framework/Parallel/CMakeLists.txt`?